### DISCUSSION TITLE
AlDente 1.22.3

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1068] [General]
**Fettuccine**

- [All] New Backend that improves stability and our ability to debug and support new MacBooks.
- [All] Fixed a bug that caused the charge limit not to work on certain systems.
- [All] Simplified the "Menubar Icon" settings.
- [All] Reduced background CPU usage.
- [All] Added Romanian language.
- [Pro] The menubar icon now updates faster.
- [Pro] Fixed a bug that caused the Schedule tab to crash under certain conditions.
- [Pro] Fixed a bug that caused the system to not go to sleep immediately after the charge limit is reached when "Disable Sleep until Charge Limit" is activated.
- [Pro] Fixed a bug that caused a glitch when the "Show Dock Icon" option was toggled too fast.
- [Pro] Power Flow now works on all supported systems.
- [Pro] Small changes and improvenemts to the schedule feature.

With peace & love from Vienna,  
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.22.3'>AlDente 1.22.3</a>.</em>

**Comment by Phate0001** [comment]


**Reply by MatthiasKerbl** [reply]
Hi @Phate0001,

you can find the support email address for AlDente Pro users in your purchase emails when you bought AlDente Pro.

Best,
Matthias

**Reply by Phate0001** [reply]
What should I do if I do not have the purchase mail anymore? I only have the invoice from paddle (Order Number / Receipt: #36445732). This is also the reason why I write here on github. I cannot connect the support because I do not have an email-adress!

**Reply by MatthiasKerbl** [reply]
You can send a request to Paddle our merchant of record at help@paddle.com to resend the purchase confirmation e-mails again.

Best,
Matthias
